libraries/lib-numeric-formats/NumericConverter.cpp

```diff
@@ -676,6 +270,37 @@ void NumericConverter::Decrement(int focusedDigit)
    Adjust(1, -1, focusedDigit);
 }
 
+bool NumericConverter::UpdateFormatter()
+{
+   if (!mFormatSymbol.empty())
+   {
+      auto formatterItem = NumericConverterRegistry::Find(mType, mFormatSymbol);
+
+      if (formatterItem == nullptr)
+      {
+         assert(formatterItem != nullptr);
+         return false;
+      }
+
+      mFormatter = formatterItem->factory({ mSampleRate });
+   }
+   else if (!mCustomFormat.empty ())
+   {
+      auto formatter = CreateParsedNumericConverterFormatter(
+         mType, mCustomFormat.Translation(), mSampleRate);
```

Comment:
Repeated work when you come here from `SetCustomFormat`, but there are other paths